client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -395,7 +395,7 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
     }
 
     const getFormattedDate = (date: string) => {
-        return format(new Date(date), 'dd/MM')
+        return format(new Date(date.split('T')[0]), 'dd/MM')
```

Comment:
maybe 'T' can be a const? can explain it better

Reply:
and maybe export new Date(date.split('T')[0]) to a function

Reply:
seperated to more verbose `truncateDate` [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1124/commits/c531470dd074f440425d0062302e18bde3deb950)